pkg/operator/edgefs/cluster/controller.go

```diff
@@ -322,7 +322,10 @@ func (c *ClusterController) handleUpdate(newClust *edgefsv1beta1.Cluster, cluste
 }
 
 func (c *ClusterController) onDelete(obj interface{}) {
-	clust := obj.(*edgefsv1beta1.Cluster).DeepCopy()
+	clust, ok := obj.(*edgefsv1beta1.Cluster)
+	if ok {
+		clust = clust.DeepCopy()
+	}
 
 	logger.Infof("delete event for cluster %s in namespace %s", clust.Name, clust.Namespace)
```

Comment:
If the type assertion failed, we will try to access the invalid `clust` object. Did you intend to return if it failed?

Reply:
fixed